santoku/google/__init__.py

```diff
@@ -1,6 +0,0 @@
-from ..google.bigquery import BigQueryHandler
-
-"""
```

Comment:
It seems okay to leave the docstring here. Will likely help in the future when we finally implement sphinx I suppose. Only change the word module for subpackage.